databases/week3/sql_week3.sql

```diff
@@ -0,0 +1,237 @@
+-- Delete a reservation with any id, fx 1
+DELETE 
+FROM reservation
+WHERE id=6;
+
+-- Queries for Review
+SELECT review.stars
+FROM review;
+
+-- Add a new review
+ INSERT INTO review(title, description, meal_id, stars)
+ VALUES ('Food for soul', 'Had delicious food with the best host ', 9, 5);
+
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE review
+SET meal_id=11
+WHERE id=6;
+
+-- Delete a review with any id, fx 1
+DELETE 
+FROM review
+WHERE id=4;
+ 
+ -- Additional queries
+ 
+ INSERT INTO meal (title, description, location, event_day, max_reservations,price, created_date)
+VALUES
+('Neapolitan Pizza', 'Neapolitan pizza, or pizza Napoletana, is a type of pizza that originated in Naples, Italy. This style of pizza is prepared with simple and fresh', 'Copenhagen', '2021-07-15 11:30:00', 10, 80, now()),
+('Thai Red Curry', ' Thai Red curry is a popular Thai dish consisting of red curry paste cooked in coconut milk with meat added, such as chicken, beef, pork, duck or shrimp, or vegetarian protein source such as tofu.', 'Roskilde', '2021-07-13 12:30:00', 12, 130, now()),
+('Vegetable Stew', ' Vegetable Stew is a combination of vegetable creamy curry prepared with thick and thin coconut milk and dry spices and served with rice', 'Valby', '2021-07-16 18:30:00', 12, 120, now()),
+('Chicken Kadai ', 'Kadai chicken is a delicious, spicy & flavorful dish made with chicken, onions, tomatoes, ginger, garlic & fresh ground spices known as kadai masala.', 'Frederiksberg', '2021-07-14 18:00:00',12 , 110, now());
+
+
+INSERT INTO reservation(number_of_guests, created_date, contact_phonenumber, contact_name, contact_email, meal_id)
+ VALUES (5, now(), '91352857','Dumbell Shiro', 'dshiro@spotify.com', 15),
+ (4, now(), '89152968','Guruwain Kumar', 'gKumar@phoca.cz', 16),
+ (3, now(), '67456795','Kyle Bessler',  'kbessler@chronoengine.com',17),
+ (2, now(), '34329630','Turdy Ray',  'tRay@harvard.edu', 18);
+ 
+INSERT INTO reservation(number_of_guests, created_date, contact_phonenumber, contact_name, contact_email, meal_id)
+ VALUES (9, now(), '91358597','Biju Sambhar', 'bsambhar@spotify.com', 17);
+ 
+
+
+ INSERT INTO review(title, description, meal_id, stars)
+ VALUES ('Excellent food', 'What a lovely homely expereince. It was food for my soul ', 15, 5), 
+ ('Will do it again', 'That was some lovley food: i recommend everone to try it out ', 16, 4), 
+ ('Yummy food', 'It was good, but not the best ', 17, 3),
+  ('It was average', 'It was average. The food had no flavour ', 17, 2);
+  
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT meal.title, meal.price
+FROM meal
+WHERE price<90;
+
+-- Get meals that still has available reservations
+SELECT meal.id, meal.title, meal.max_reservations AS max_guests, SUM(reservation.number_of_guests) AS total_reserved
+FROM meal
+INNER JOIN reservation ON meal.id = reservation.meal_id
+GROUP BY meal.title
+HAVING total_reserved < max_guests;
```

Comment:
This only considers the meals that have at least one reservation. How would you also get the meals without any reservations?

Reply:
I tried this, but it still doesn't solve the issue

Reply:
I can give a hint and say that you should try out a different type of join :)